geoapps/inversion/line_sweep/driver.py

```diff
@@ -39,20 +39,21 @@ def run(self):  # pylint: disable=W0221
 
     def setup_params(self):
         h5_file_path = Path(self.workspace.h5file).resolve()
-        ui_json_path = h5_file_path.with_suffix(".ui.json")
-        if not ui_json_path.is_file():
+        ui_json_path = h5_file_path.stem.replace(".ui", "") + ".ui.json"
+        if not (h5_file_path / ui_json_path).is_file():
```

Comment:
**suggestion (naming):" either rename to `ui_json_name` or `ui_json_path` with the full path this way:
```suggestion
        ui_json_path = h5_file_path.parent / h5_file_path.stem.replace(".ui", ".ui.json")
        if not (ui_json_path).is_file():
```